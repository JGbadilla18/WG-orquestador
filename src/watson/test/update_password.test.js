const testWatson = require('./testWatson');

watsonText = 'Actualizar mi contraseÃ±a.'

it(watsonText, () => {
  expect.assertions(1);
  return testWatson(watsonText).then(data => expect(data).toMatchSnapshot());
});
