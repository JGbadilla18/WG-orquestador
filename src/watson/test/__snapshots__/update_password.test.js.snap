// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Actualizar mi contraseña. 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "response_type": "text",
      "text": "Puedes reiniciar tu contraseña de Bi en Línea ingresando a la opción \\"Olvidé mi contraseña\\" desde la web o App. En caso no tengas tu número celular actualizado en el perfil de seguridad, puedes ingresar a nuestro Whatsapp Bi 2411-6000.",
    },
    Object {
      "elements": Array [
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más aquí",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/reinicio-de-contrasena",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/04/Reinicio-contraseña-BEL.jpg",
          "subtitle": "Aprende cómo reiniciar tu contraseña.",
          "title": "Reinicio de contraseña desde Bi en Línea",
        },
      ],
      "response_index": 1,
      "response_type": "carousel",
    },
    Object {
      "response_type": "pause",
      "time": 10000,
      "typing": true,
    },
  ],
}
`;
